martin-mbtiles/src/tile_copier.rs

```diff
@@ -98,23 +104,35 @@ impl TileCopier {
             .execute(&mut conn)
             .await?;
 
-        let schema = query("SELECT sql FROM sourceDb.sqlite_schema WHERE tbl_name IN ('metadata', 'tiles', 'map', 'images')")
-            .fetch_all(&mut conn)
-            .await?;
-
-        for row in &schema {
-            let row: String = row.get(0);
-            query(row.as_str()).execute(&mut conn).await?;
-        }
+        if self.options.force_simple {
```

Comment:
another thing - let's use force simple only if we detect that the source is a deduplicated mbtiles